python/tvm/relay/frontend/keras.py

```diff
@@ -663,6 +667,36 @@ def _convert_padding(inexpr, keras_layer, etab):
         return _op.nn.pad(data=inexpr, pad_width=((0, 0), (0, 0), (top, bottom), (left, right)))
     return _op.nn.pad(data=inexpr, pad_width=((0, 0), (top, bottom), (left, right), (0, 0)))
 
+def _convert_padding3d(inexpr, keras_layer, etab):
+    _check_data_format(keras_layer)
+    padding = keras_layer.padding
+
+    d_pad = h_pad = w_pad = {0, 0}
+    if isinstance(padding, tuple):
```

Comment:
It seems _convert_padding3d only supports the case when keras_layer.padding is 3 tuples of 2 ints. Is this true?

Reply:
`test_forward_zero_padding3d` i have added this 3 scenarios in test scripts and all 3 are ok. keras will send as 3 tuples of 2 ints in any scenario.

Reply:
Thanks for the clarification. Then the two checks can be merged into one: `if isinstance(padding, tuple) and isinstance(padding[0], tuple)`

Reply:
I also suggest adding a comment to make it clear that keras will send as 3 tuples of 2 ints in all three scenarios.